### PR TITLE
[WG/IWWG] fixup draft for targetting HR and PROPOSED

### PR DESCRIPTION
**Pull request by himorin**
@Yonet @AdaRoseCannon @cwilso 
fixed various parts to move our draft to HR and PROPOSED for AC review. Please check and mark with thumb up or something.

one point, do we want to keep [2 implementation for CRS](https://w3c.github.io/charter-drafts/2024/immersive-web-wg.html#success-criteria) at the top of Success Criteria?

### CODE REVIEW

File: 2024/immersive-web-wg.html [hunk 1]
```diff
@@ -130,7 +130,7 @@ <h1 id="title">Immersive Web Working Group Charter</h1>
             Chairs
           </th>
           <td>
-            Ada Rose Cannon (Samsung), Chris Wilson (Google), Ayşegül Yönet (ARtistus)
+            Ada Rose Cannon (Samsung), Chris Wilson (Google), Ayşegül Yönet (W3C Invited Experts)
```
Comment (cwilso) [inline 2024/immersive-web-wg.html]:
```suggestion
            Ada Rose Cannon (Samsung), Chris Wilson (Google), Ayşegül Yönet (W3C Invited Expert)
```

**Review by cwilso — APPROVED** [review]

File: 2024/immersive-web-wg.html (hunk 1 above)
Comment (himorin) [reply 2024/immersive-web-wg.html]: really thank you for catching this!

**Comment by himorin** [comment]
please note, if no comment provided on following part,
> one point, do we want to keep [2 implementation for CRS](https://w3c.github.io/charter-drafts/2024/immersive-web-wg.html#success-criteria) at the top of Success Criteria?

I will just remove `todo` and make that part remaining in HR and PROPOSED version.

**Comment by AdaRoseCannon** [comment]
I think we should keep it personally.

**Comment by himorin** [comment]
updated per comments. ready for merge and start review.

**Pull request merged**